src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,75 @@
+package homework_2.random_chars_table;
+
+import homework_1.Run;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class RandomCharsTable implements Run {
```

Comment:
Has unnesessary whitespaces in strategy output
![image](https://user-images.githubusercontent.com/58872509/126910821-5038a0a8-2b6f-4dd5-87bf-14a525187f39.png)